ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviderForm.tsx

```diff
@@ -482,53 +486,123 @@ function AuthProviderForm({
-                                titleElement="h3"
-                            >
-                                {selectedAuthProvider.requiredAttributes.map(
-                                    (attribute, index: number) => (
-                                        <Flex
-                                            key={`${attribute.attributeKey}_required_attribute_${index}`}
-                                        >
-                                            <FormGroup label="Key" fieldId={attribute.attributeKey}>
-                                                <TextInput
-                                                    type="text"
-                                                    id={attribute.attributeKey}
-                                                    value={attribute.attributeKey}
-                                                    isDisabled
-                                                />
-                                            </FormGroup>
-                                            <FormGroup
-                                                label="Value"
-                                                fieldId={attribute.attributeValue}
-                                            >
-                                                <TextInput
-                                                    type="text"
-                                                    id={attribute.attributeValue}
-                                                    value={attribute.attributeValue}
-                                                    isDisabled
-                                                />
-                                            </FormGroup>
-                                        </Flex>
-                                    )
+                    {selectedAuthProvider.type === 'oidc' && (
+                        <FormSection
+                            title="Required attributes for the authentication provider"
+                            titleElement="h3"
+                        >
+                            <div id="required-attributes-description">
+                                <Alert isInline variant="info" title="">
+                                    <p>
+                                        The required attributes are used to require attributes being
+                                        returned from the authentication provider.
+                                    </p>
+                                    <p>
+                                        In case a required attribute is not returned from the
+                                        authentication provider, the login attempt will fail and no
+                                        role will be assigned to the user.
+                                    </p>
+                                </Alert>
+                            </div>
+                            {values.requiredAttributes.length === 0 && (
```

Comment:
Previously, we were using `selectedAttribute` but I noticed it changed to `values`. Why is that?

Reply:
`selectedAuthProvider` is the value that passed down, while `values` contain modifiable value attached to this form